frontend/src/HOC/AuthenticationGuard.js

```diff
@@ -39,21 +39,18 @@ export const AuthenticationGuard = (WrappedComponent) => {
     async keycloakInit() {
       // Initialize client
       const keycloak = Keycloak(KEYCLOAK);
-      await keycloak.init();
-
-      // Prompt for login using IDIR if not authenticated
-      if (!keycloak.authenticated) {
-        await keycloak.login({
+      await keycloak
```

Comment:
Doesn't look like the `await` here does anything 😄

Reply:
I'm inclined to agree with you :P. I'll remove it.

Reply:
To follow-up, removing an await within an async shouldn't change the behaviour...but it caused the application to get caught in a login loop. Changing the keycloakInit to a synchronous function fixes it, but I'm not sure if thats the behaviour we want.

Reply:
Promises are non-blocking, so a "non-async" function is totally fine here, if that's your concern